workers/ohsome_quality_analyst/geodatabase/client.py

```diff
@@ -180,9 +180,12 @@ async def get_feature_from_db(dataset: str, feature_id: str) -> Feature:
     return Feature(geometry=geojson.loads(result[0]))
 
 
-async def get_available_regions() -> FeatureCollection:
+async def get_available_regions(with_geometries: bool) -> FeatureCollection:
```

Comment:
I think it does not make sense to return a GeoJSON object without the geometries (Which makes it invalid).

I suggest to implement two function: `get_regions_as_geojson() -> FeatureCollection` and `get_regions() -> List[Tuple[str, str]]`. The second one will only get names and feature_id (ogc_fid) from the databse and will not create a GeoJSON. 

What do you think about this?